### DISCUSSION TITLE
Have you any plan to continue activ delevlopment of Gaguge

### DISCUSSION THREAD

**Discussion by dorion** [getgauge/gauge#2277] [General]
Is there hope to continue the active development or should we look for alternative applications to Gauge?
It would be a pity if not, because it is a great E2E tool for the BDD. I don't like the plain Gherkin language.

**Comment by zabil** [comment]
We talk about that here
https://github.com/getgauge/gauge/issues/1732

**Reply by johnboyes** [reply]
In #1732 you mentioned that you were/are actively looking to make Gauge part of a foundation, @zabil.  As it's now been 2 years, what's the latest, is that (making Gauge part of a foundation) still the goal?  If not, is the future of Gauge viable on its current course and speed?  Really hope so as its a fantastic tool 🙏🏻 
If there's any actions that you'd like help on then please share here.

**Reply by zabil** [reply]
Hey @johnboyes 
Thanks for asking about that. We spoke to folks at SF conservancy the Selenium as well as the JS foundation. These foundations looked at our activity and recommended that we are not ready yet. 

Most of these projects that transitioned to foundations spent close to 10 years evolving on their own before they joined. So the best way now is to support it via Github till there's a big enough community so that it can be managed via a foundation.
